include/picovoice.h

```diff
@@ -77,7 +85,14 @@ PV_API pv_status_t pv_get_error_stack(
  */
 PV_API void pv_free_error_stack(char **message_stack);
 
+PV_API void pv_set_sdk(const char *sdk);
+
+PV_API const char *pv_get_sdk(void);
+
+PV_API void pv_free(void *p);
+
```

Comment:
we're not adding these to the public header, we should be using the clean one with onlye the `_MSC_VER` change